src/styled/customStyles.scss

```diff
@@ -1,5 +1,13 @@
+// home custom styles
+.home-affilications,
```

Comment:
is affilications suppose to be affiliations?

Reply:
dont know why my finger keeps hitting the c